.gitignore

```diff
@@ -4,6 +4,9 @@
 # no longer check-in docs for builds.  This is handled by Netlify
 /docs
 
+# tsc checks build here
```

Comment:
With the introduction of `noEmit: true`, this pattern addition is no longer necessary.